### PR TITLE
[12.0][MIG] stock_orderpoint_move_link

### PR DESCRIPTION
**Pull request by kittiu**
@jbeficent 
I have migrated based on how I see the code changes in v12. Frankly, I am not very sure what it all about functionally, if you would explain, I can continue with test script.

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@kittiu The reason for this module is that you want to understand what orderpoint originated the move. And you need that because ultimately you want to provide visibility in the Purchase Order or Manufacturing order of the status of the orderpoint.

**Comment by kittiu** [comment]
@jbeficent ok Jordi. By the way, I found that module stock_orderpoint_purchase_link is related to this and it does have test already.
Should I do both stock_orderpoint_move_link and stock_orderpoint_purchase_link in the same PR?

**Comment by kittiu** [comment]
@jbeficent, change my mind, I will stock_orderpoint_purchase_link separately. Let me finish that one first, it will make more sense to me, and you can review both later. OK?

File: stock_orderpoint_move_link/__manifest__.py
Comment (LoisRForgeFlow) [inline stock_orderpoint_move_link/__manifest__.py]: A migration does not make you author of a module, you should put yourself in the contributors fragment of the readme.

File: stock_orderpoint_move_link/README.rst
Comment (LoisRForgeFlow) [inline stock_orderpoint_move_link/README.rst]: better try to move the README to a fragmented version: follow the template here: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Review by LoisRForgeFlow — COMMENTED** [review]
Hi @kittiu 

thanks for the work. See some small comments inline that may apply to all migration you do.

BTW, the PRs must be done always one module at a time, and specify dependecies to other open PR if that's the case.

File: stock_orderpoint_move_link/README.rst
Comment (kittiu) [reply stock_orderpoint_move_link/README.rst]: Thank you @lreficent will let you guys know again once it is ready, also together with purchase_move_link.

Comment (LoisRForgeFlow) [reply stock_orderpoint_move_link/README.rst]: let me know if you have any doubt while fragmenting the readme

Comment (kittiu) [reply stock_orderpoint_move_link/README.rst]: @lreficent I now understand how the readme works. But I have other question.

I did migrate to v12 (on my local) the **stock_orderpoint_purchase_link**, also can test it OK.
But still I don't get to test **stock_orderpoint_move_link** (even in v11). Can you hint me the test scenario? and I also can add the test script.

As you suggest, I can migrate **stock_orderpoint_purchase_link** later once I done with the first one, since it is dependency.

Comment (LoisRForgeFlow) [reply stock_orderpoint_move_link/README.rst]: You can test it enabling the reception in 2 or 3 steps and then running an orderpoint, moves should have the link to the orderpoint that generated them.

Remember: You still have fill in the fragment of the README, you now only have the template in place.

Comment (kittiu) [reply stock_orderpoint_move_link/README.rst]: Thank you @lreficent , I have tested and done the README. Please let me know if I need anything else.

**Comment by kittiu** [comment]
@lreficent for this module, I think we can not write test case, otherwise we would need to depends on also purchase module (so we can test). What do you think?

I submitted stock_orderpoint_purchase_link but that one need to depend on this to pass the test.

Next, I am looking into doing stock_orderpoint_manual_procurement (as part of DDMRP), WDYT?

**Comment by LoisRForgeFlow** [comment]
@kittiu I made you a PR including test: https://github.com/kittiu/stock-logistics-warehouse/pull/1 Could you please have a look?

**Review by LoisRForgeFlow — APPROVED** [review]
Great, it LGTM now. Just one small extra thing and we are ready to merge, can you squash your 2 migration commits into one? the one that says "in progress - readme..." can be included in the migration to v12 one.

Thanks for the work!

**Review by rousseldenis — APPROVED** [review]
Code review. LGTM.

**Comment by rousseldenis** [comment]
@kittiu @lreficent One remark: the method '_merge_moves_fields' doesn't seem to be tested.

**Comment by LoisRForgeFlow** [comment]
@rousseldenis Do you find it required to test that method to move forward?

**Comment by rousseldenis** [comment]
@lreficent Nope. That should avoid future regressions but not blocking. Let's merge

**Comment by LoisRForgeFlow** [comment]
@kittiu I just did the squash myself so we can move forward now. I'll merge after the CI passes. 

Please, you will be able to rebase the PR with dependencies to this module after the merge.

**Pull request merged**
